pkg/types/args_test.go

```diff
@@ -0,0 +1,98 @@
+package types_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/appc/cni/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+func TestArgs(t *testing.T) {
```

Comment:
I think I got it. I ran `ginkgo bootstrap` in the `pkgs/types` directory. Please let me know if we're good now!